cm.json/src/test/java/org/apache/felix/cm/json/io/impl/JsonSupportTest.java

```diff
@@ -307,4 +307,19 @@ public void testEscapedQuotes() throws IOException {
                 + "}\n", parse(input));
     }
 
+    @Test
+    public void testSlashAtEndOfRead() throws Exception {
+        String input = "This is a test string // Next string \n"
+                + "next line" ;
+        Reader reader = JsonSupport.createCommentRemovingReader(new StringReader(input));
+
+        char[] buffer = new char[22];
```

Comment:
Nit: Instead of the magic number 22, how about using `input.indexOf('/')`? Or something equivalent, if I misunderstood the intent. IMHO that would make it clearer what you're testing.